Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -2071,7 +2132,7 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
         discoveryGroup.async(queue: discoveryQueue) { [unowned self] in
             symbolGraphLoader = SymbolGraphLoader(
                 bundle: bundle,
-                dataProvider: self.dataProvider,
+                dataLoader: { try self.contentsOfURL($0, in: $1) },
```

Comment:
Why is this a call to a class method? Where is that defined? I only see `contentsOfURL(_:in:)` above written as an instance method?

Reply:
Oops I see it's not a class method. You have lowercase `self`